<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd"> <html><body> <div class="publications"> <h2 class="bibliography">2024</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="10.1007/978-1-0716-3989-4_52" class="col-sm-8"> <div class="title">A Scalable Adaptive Quadratic Kernel Method for Interpretable Epistasis Analysis in Complex Traits</div> <div class="author"> Boyang Fu* , <em>Prateek Anand*</em>, Aakarsh Anand* , Joel Mefford , and Sriram Sankararaman </div> <div class="periodical"> <em>In Research in Computational Molecular Biology</em> , 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://link.springer.com/chapter/10.1007/978-1-0716-3989-4_52" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>Our knowledge of the contribution of genetic interactions (epistasis) to variation in human complex traits remains limited, partly due to the lack of efficient, powerful, and interpretable algorithms to detect interactions. Recently proposed approaches for set-based association tests show promise in improving power to detect epistasis by examining the aggregated effects of multiple variants. Nevertheless, these methods either do not scale to large numbers of individuals available in Biobank datasets or do not provide interpretable results. We, therefore, propose QuadKAST, a scalable algorithm focused on testing pairwise interaction effects (also termed as quadratic effects) of a set of genetic variants on a trait and quantifying the proportion of phenotypic variance explained by these effects. We performed comprehensive simulations and demonstrated that QuadKAST is well-calibrated with good statistical power. We applied QuadKAST to 53 quantitative phenotypes measured in }}\backslashapprox 300,000}}≈300,000unrelated white British individuals in the UK Biobank to test for quadratic effects within each of }}9,515}}9,515protein-coding genes (after accounting for linear additive effects). We detected }}32}}32trait-gene pairs across }}17}}17traits that demonstrate statistically significant signals of quadratic effects (}}p \backslashle \backslashfrac{0.05}{9,515\backslashtimes 53}}}p≤0.059,515\texttimes53accounting for the number of genes and traits tested). Our method enables the detailed investigation of epistasis on a large scale, offering new insights into its role and importance.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">2023</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> </div> <div id="Fu2023.09.10.557084" class="col-sm-8"> <div class="title">A biobank-scale test of marginal epistasis reveals genome-wide signals of polygenic epistasis</div> <div class="author"> Boyang Fu* , Ali Pazokitoroudi* , Albert Xue , Aakarsh Anand , <em>Prateek Anand</em>, Noah Zaitlen , and Sriram Sankararaman </div> <div class="periodical"> <em>bioRxiv</em>, 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://www.biorxiv.org/content/10.1101/2023.09.10.557084v1" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> </div> <div class="abstract hidden"> <p>The contribution of epistasis (interactions among genes or genetic variants) to human complex trait variation remains poorly understood. Methods that aim to explicitly identify pairs of genetic variants, usually single nucleotide polymorphisms (SNPs), associated with a trait suffer from low power due to the large number of hypotheses tested while also having to deal with the computational problem of searching over a potentially large number of candidate pairs. An alternate approach involves testing whether a single SNP modulates variation in a trait against a polygenic background. While overcoming the limitation of low power, such tests of polygenic or marginal epistasis (ME) are infeasible on Biobank-scale data where hundreds of thousands of individuals are genotyped over millions of SNPs.We present a method to test for ME of a SNP on a trait that is applicable to biobank-scale data. We performed extensive simulations to show that our method provides calibrated tests of ME. We applied our method to test for ME at SNPs that are associated with 53 quantitative traits across ≈ 300 K unrelated white British individuals in the UK Biobank (UKBB). Testing 15, 601 trait-loci associations that were significant in GWAS, we identified 16 trait-loci pairs across 12 traits that demonstrate strong evidence of ME signals (p-value ). We further partitioned the significant ME signals across the genome to identify 6 trait-loci pairs with evidence of local (within-chromosome) ME while 15 show evidence of distal (cross-chromosome) ME. Across the 16 trait-loci pairs, we document that the proportion of trait variance explained by ME is about 12x as large as that explained by the GWAS effects on average (range: 0.59 to 43.89). Our results show, for the first time, evidence of interaction effects between individual genetic variants and overall polygenic background modulating complex trait variation.Competing Interest StatementThe authors have declared no competing interest.</p> </div> </div> </div> </li></ol> <h2 class="bibliography">1935</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <abbr class="badge"><a href="https://journals.aps.org/" rel="external nofollow noopener" target="_blank">PhysRev</a></abbr> </div> <div id="PhysRev.47.777" class="col-sm-8"> <div class="title">A scalable adaptive quadratic kernel method for interpretable epistasis analysis in complex traits</div> <div class="author"> A. Einstein , B. Podolsky , and N. Rosen </div> <div class="periodical"> <em>Phys. Rev.</em>, May 1935 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abs</a> <a href="https://journals.aps.org/pr/abstract/10.1103/PhysRev.47.777" class="btn btn-sm z-depth-0" role="button" rel="external nofollow noopener" target="_blank">HTML</a> <a href="/assets/pdf/example_pdf.pdf" class="btn btn-sm z-depth-0" role="button">PDF</a> </div> <div class="badges"> <span class="altmetric-embed" data-hide-no-mentions="true" data-hide-less-than="15" data-badge-type="2" data-badge-popover="right" data-doi="10.1103/PhysRev.47.777"></span> <span class="__dimensions_badge_embed__" data-doi="10.1103/PhysRev.47.777" data-hide-zero-citations="true" data-style="small_rectangle" data-legend="hover-right" style="margin-bottom: 3px;"></span> <a href="https://scholar.google.com/citations?view_op=view_citation&amp;hl=en&amp;user=qc6CJjYAAAAJ&amp;citation_for_view=qc6CJjYAAAAJ:qyhmnyLat1gC" aria-label="Google Scholar link" role="button" rel="external nofollow noopener" target="_blank"> <img src="https://img.shields.io/badge/scholar-25K-4285F4?logo=googlescholar&amp;labelColor=beige" alt="25K Google Scholar citations"> </a> </div> <div class="abstract hidden"> <p>In a complete theory there is an element corresponding to each element of reality. A sufficient condition for the reality of a physical quantity is the possibility of predicting it with certainty, without disturbing the system. In quantum mechanics in the case of two physical quantities described by non-commuting operators, the knowledge of one precludes the knowledge of the other. Then either (1) the description of reality given by the wave function in quantum mechanics is not complete or (2) these two quantities cannot have simultaneous reality. Consideration of the problem of making predictions concerning a system on the basis of measurements made on another system that had previously interacted with it leads to the result that if (1) is false then (2) is also false. One is thus led to conclude that the description of reality as given by a wave function is not complete.</p> </div> </div> </div> </li></ol> </div> </body></html>